src/Notifier.php

```diff
@@ -41,7 +44,10 @@ public function __construct($opt = [])
 
         $this->opt = array_merge([
             'host' => 'api.airbrake.io',
+            'rootDirectory' => null,
```

Comment:
Is this needed? It catches attention because this is the only option with null value.